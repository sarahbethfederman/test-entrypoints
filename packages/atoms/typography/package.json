{
  "name": "@auscred/typography",
  "version": "0.0.1",
  "main": "dist/Typography.js",
  "types": "dist/Typography.d.ts",
  "files": [
    "dist"
  ],
  "keywords": [],
  "author": "Lendi Devs",
  "license": "ISC",
  "peerDependencies": {
    "@types/react": "^16.4.0",
    "react": "^16.4.0",
    "styled-components": "^3.3.3"
  },
  "scripts": {
    "build": "tsc --pretty"
  }
}
