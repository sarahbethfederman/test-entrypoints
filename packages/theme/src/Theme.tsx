import * as React from 'react';
import styled, { ThemeProvider, injectGlobal } from 'styled-components';
import { theme } from './theme/lendi';
import { ThemeMap } from './types';

interface ThemeProps {
  kind?: ThemeMap;
  children?: React.ReactNode;
}

// Any cross browser resets should occur here
// tslint:disable-next-line no-unused-expression
injectGlobal`
  html, body {
    margin: 0;
    padding: 0;
  }

  :root {
    --lendi-ui-size: 14px;
  }

  @media (min-width: 36.0625em) {
    :root {
      --lendi-ui-size: 16px;
    }
  }
`;

const Root = styled.div`
  color: ${theme.colors.shade[700]};
`;

export function Theme(props: ThemeProps) {
  const { children } = props;
  const kind = props.kind || theme;

  return (
    <ThemeProvider theme={kind}>
      <Root>{children}</Root>
    </ThemeProvider>
  );
}
