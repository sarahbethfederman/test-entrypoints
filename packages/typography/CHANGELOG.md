## [4.1.0] 2020-06-24 [Release Notes]
- Added 'xl' as a size for Body

## [4.0.1] 2019-09-26 [Release Notes](https://creditandfinance.atlassian.net/wiki/spaces/HUB/pages/803930391/Upcoming+Major+Changes)
- Bumped lendi-ui dependencies to latest versions

## [4.0.0] 2019-9-23 2019-8-21 [Release notes](https://creditandfinance.atlassian.net/wiki/spaces/HUB/pages/803930391/Upcoming+Major+Changes)
### Changed
- Uses Theme v7
- Update typescript to version 3.5.2 and react to version 16.8.4 [HUB-151](https://creditandfinance.atlassian.net/browse/HUB-151)

## [3.4.0] - 2019-07-29 - [HUB-338](https://creditandfinance.atlassian.net/browse/HUB-338)
### Added
- Add 'xxs' type of Body, Heading, Link and Overline.

## [3.3.0] - 2019-07-16 - [HUB-330](https://creditandfinance.atlassian.net/browse/HUB-330)
### Changed
- Change link default color to 'currentColor'

## [3.2.1] - 2019-06-4 - [HUB-262](https://creditandfinance.atlassian.net/browse/HUB-262)
### Changed
- Updated utils dependency to get correct types

## [3.2.0] - 2019-05-27 - [HUB-169](https://creditandfinance.atlassian.net/browse/HUB-169)
### Changed
- Support for native props like Aria attributes, Standard HTML Attributes like title, classname, id, role, itemProp, itemID, itemRef

## [3.1.1] - 2019-05-23 - [HUB-201](https://creditandfinance.atlassian.net/browse/HUB-201)
### Changed
- Updated links that are set as a button to inherit background-color from parent.

## [3.1.0] - 2019-05-17 - [HUB-174](https://creditandfinance.atlassian.net/browse/HUB-174)
### Changed
- Module format reverted back to CJS.
### Deleted
- Dropped "module" property from package.json.

## [3.0.2] - 2019-05-09 - [HUB-92](https://creditandfinance.atlassian.net/browse/HUB-92)
### Changed
- Updated CHANGELOG to match required parsing format.

## [3.0.1] - 2019-05-08 - [HUB-54](https://creditandfinance.atlassian.net/browse/HUB-54)
### Changed
- Change module main entry point to esm in package.json

## [3.0.0] - 2019-04-30 - [HUB-54](https://creditandfinance.atlassian.net/browse/HUB-54)
### Deleted
- Remove the CJS bundles.

## [2.7.1] - 2019-04-10 - [HUB-50](https://creditandfinance.atlassian.net/browse/HUB-50)
### Changed
- Fixed the margin bug for Overline and added some tests for margin alignment in all typography.

## [2.7.0] - 2019-04-10 - [HUB-90](https://creditandfinance.atlassian.net/browse/HUB-90)
### Added
- Added "xl" size variant.

## [2.6.0] - 2019-03-14 - [BRAVO-571](https://creditandfinance.atlassian.net/browse/BRAVO-571)
### Added
- Added `Overline` sub-package, which includes `overline` mixin and `Overline` component.

## [2.5.5] - 2019-03-07 - [BRAVO-652](https://creditandfinance.atlassian.net/browse/BRAVO-652)
### Changed
- fixed bug that caused the error: 'children' doesn't exist on type 'IntrinsicAttributes & LinkProps'
