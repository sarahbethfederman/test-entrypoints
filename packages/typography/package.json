{
  "name": "@lendi-ui/typography",
  "version": "6.0.0-beta.0",
  "source": "src/index.ts",
  "main": "dist/typography.cjs.js",
  "module": "dist/typography.esm.js",
  "files": [
    "dist"
  ],
  "license": "ISC",
  "peerDependencies": {
    "@lendi-ui/theme": "^10.0.0-beta.0",
    "@types/react": "^16.8",
    "@types/styled-components": "^4.1.18",
    "react": "^16.8",
    "styled-components": "^4"
  },
  "dependencies": {
    "@lendi-ui/breakpoint": "^6.0.0-beta.0",
    "@lendi-ui/color": "^6.0.0-beta.0",
    "@lendi-ui/spacing": "^7.1.0-beta.0",
    "@lendi-ui/utils": "^6.0.0-beta.0"
  },
  "devDependencies": {
    "@lendi-ui/theme": "10.0.0-beta.0",
    "@types/react": "^16.4.8",
    "@types/styled-components": "^4.1.18",
    "react": "^16.9.0",
    "styled-components": "^4"
  }
}
