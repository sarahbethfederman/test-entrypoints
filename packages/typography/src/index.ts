import styled, { css } from 'styled-components';

import { select } from '@lendi-ui/theme';
import { gte, map, BreakpointValue, BreakpointValueMap } from '@lendi-ui/breakpoint';
import { fg } from '@lendi-ui/color';

export type Alignment = 'left' | 'center' | 'right' | 'justify';
export type AlignmentOrAlignmentMap = BreakpointValue<Alignment> | BreakpointValueMap<Alignment>;
export type HeadingSize = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
export type BodySize = 'xs' | 'sm' | 'md' | 'lg';
export type LinkSize = 'sm' | 'md' | 'lg';

const alignMixin = (alignment: AlignmentOrAlignmentMap) => {
  return map(alignment, (val) => {
    if (val) {
      return `text-align: ${val};`;
    } else {
      return '';
    }
  });
};

const headingSizeMixin = (size: HeadingSize) => {
  switch (size) {
    case 'xs':
      return css`
        font-size: 15px;
        font-weight: 500;
        line-height: calc(20 / 15);

        ${gte('tablet')`
          font-size: 18px;
          line-height: calc(24/18);
        `};
      `;
    case 'sm':
      return css`
        font-size: 18px;
        font-weight: 500;
        line-height: calc(24 / 18);

        ${gte('tablet')`
          font-size: 22px;
          line-height: calc(28/22);
        `};
      `;
    case 'md':
      return css`
        font-size: 22px;
        font-weight: 500;
        line-height: calc(28 / 22);

        ${gte('tablet')`
          font-size: 28px;
          line-height: calc(32/28);
        `};
      `;
    case 'lg':
      return css`
        font-size: 28px;
        font-weight: 500;
        line-height: calc(34 / 28);

        ${gte('tablet')`
          font-size: 37px;
          line-height: calc(40/37);
        `};
      `;
    case 'xl':
      return css`
        font-size: 37px;
        font-weight: bold;
        line-height: calc(40 / 37);

        ${gte('tablet')`
          font-size: 46px;
          line-height: calc(50/46);
        `};
      `;
  }
};

const bodySizeMixin = (size: BodySize) => {
  switch (size) {
    case 'xs':
      return css`
        font-size: 10px;
        line-height: calc(16 / 10);

        ${gte('tablet')`
          font-size: 13px;
          line-height: calc(20/13);
        `};
      `;
    case 'sm':
      return css`
        font-size: 12px;
        line-height: calc(16 / 12);

        ${gte('tablet')`
          font-size: 14px;
          line-height: calc(20/14);
        `};
      `;
    case 'md':
      return css`
        font-size: 14px;
        line-height: calc(20 / 14);

        ${gte('tablet')`
          font-size: 16px;
          line-height: calc(24/16);
        `};
      `;
    case 'lg':
      return css`
        font-size: 16px;
        line-height: calc(24 / 16);

        ${gte('tablet')`
          font-size: 18px;
          line-height: calc(24/18);
        `};
      `;
  }
};

export interface HeadingOptions {
  color?: string;
  align?: Alignment;
  size: HeadingSize;
}

export function heading(options: HeadingOptions) {
  const { color = 'shade.700', align, size } = options;
  return css`
    ${color && fg(color)}
    ${align && alignMixin(align)}
    ${headingSizeMixin(size)}
    font-family: ${select('typography.heading.fontFamily', '"Cabin", sans-serif')};
  `;
}

export interface BodyOptions {
  color?: string;
  align?: Alignment;
  size?: BodySize;
}

/**
 * Body mixin
 */
export function body(options: BodyOptions = {}) {
  const { color = 'shade.700', align, size = 'md' } = options;
  return css`
    ${color && fg(color)}
    ${align && alignMixin(align)}
    ${size && bodySizeMixin(size)}
    font-family: ${select('typography.body.fontFamily', '"Open sans", sans-serif')};
  `;
}

/**
 * Body component
 */
export const Body = styled.p`
  ${link};
`;

export interface LinkOptions {
  color?: string;
  href?: string;
  size?: LinkSize;
}

/**
 * Link mixin
 */
export function link(options: LinkOptions = {}) {
  const { color = 'primary.500', size } = options;

  return css`
    ${color && fg(color)}
    ${size && bodySizeMixin(size)}
    font-family: ${select('typography.body.fontFamily', '"Open sans", sans-serif')};
    font-weight: bold;
    cursor: pointer;

    &:hover,
    &:focus,
    &:active {
      text-decoration: underline;
      ${fg('primary.700')}
    }
  `;
}

/**
 * Link component
 */
export const Link = styled.a`
  ${link};
`;
