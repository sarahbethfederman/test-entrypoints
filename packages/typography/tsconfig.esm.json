{
  "extends": "../../tsconfig.esm.json",
  "compilerOptions": {
    "outDir": "./dist/esm"
  },
  "include": ["src/**/*.ts", "src/**/*.tsx"],
  "exclude": ["src/**/*.test.ts", "src/**/*.test.tsx", "src/**/*.doc.ts", "src/**/*.doc.tsx"]
}
