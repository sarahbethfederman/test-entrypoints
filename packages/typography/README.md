# @lendi-ui/typography

Typography including Body, Heading and Link.

## Installation

```
yarn add @lendi-ui/typography
```

## Usage

- Body

```jsx
import { Body } from '@lendi-ui/typography';
<Body size="lg" color="warn.500">
  The quick brown fox jumps over the lazy dog
</Body>;
```

- Heading

```jsx
import { Heading } from '@lendi-ui/typography';
<Heading size="lg" color="warn.500">
  The quick brown fox jumps over the lazy dog
</Heading>;
```

- Link

```jsx
import { Link } from '@lendi-ui/typography';
<Link>quick brown fox</Link>;
```
