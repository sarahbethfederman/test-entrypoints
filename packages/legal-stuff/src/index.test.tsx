import * as React from 'react';
import { mount } from 'enzyme';
import Theme from '@lendi-ui/theme';
import Legal from './index';
import { Wrapper, ContentContainer, BodyContainer } from './index.style';
import { Heading } from '@lendi-ui/typography';

const curYear = new Date().getFullYear();

describe('Legal', () => {
  let element;
  beforeEach(() => {
    element = mount(
      <Theme>
        <Legal />
      </Theme>
    );
  });
  it('should mount', () => {
    expect(element.find(Legal)).toHaveLength(1);
    expect(element.find('.curYear').text()).toEqual(curYear.toString());
    expect(element.find(Wrapper)).toHaveLength(1);
    expect(element.find(ContentContainer)).toHaveLength(1);
    expect(element.find(BodyContainer)).toHaveLength(7);
    expect(element.find(Heading)).toHaveLength(1);
    expect(element.find(Legal)).toMatchSnapshot();
    expect(element.find("meta[name='format-detection']").exists()).toEqual(true);
  });
  describe('test native props and Standard HTML Attributes', () => {
    let legalAttributes;
    beforeEach(() => {
      element = mount(
        <Theme>
          <Legal aria-label="testLabel" aria-describedby="info" id="testId" tabIndex={Number(1)} />
        </Theme>
      );
      legalAttributes = element.find(Legal).props();
    });

    it('should mount with Aria attributes', () => {
      expect(legalAttributes['aria-label']).toBe('testLabel');
      expect(legalAttributes['aria-describedby']).toBe('info');
    });
    it('should mount with native props like id, tabIndex', () => {
      expect(legalAttributes.id).toBe('testId');
      expect(legalAttributes.tabIndex).toBe(1);
    });
  });
});
