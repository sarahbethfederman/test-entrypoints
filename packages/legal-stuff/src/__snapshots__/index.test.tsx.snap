// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legal should mount 1`] = `
.c2 {
  margin-top: 0;
  margin-bottom: 0;
  color: rgba(255,255,255,1);
  font-size: calc(1.75 * var(--lendi-ui-size));
  font-weight: 400;
  line-height: calc(32 / 28);
  font-family: 'Cabin',sans-serif;
  margin-top: var(--lendi-ui-size);
}

.c0 {
  box-sizing: border-box;
  line-height: 1.333;
  white-space: normal;
  border-spacing: 2px;
  text-align: left;
  word-spacing: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  background-color: rgba(7,54,65,1);
}

.c1 {
  box-sizing: border-box;
  max-width: 75rem;
  width: 100%;
  margin-left: auto;
  margin-right: auto;
  height: 100%;
  padding: calc(1.5 * var(--lendi-ui-size));
}

.c3 {
  color: rgba(200,200,200,1);
  font-size: calc(0.75 * var(--lendi-ui-size));
  line-height: calc(16 / 12);
  font-family: 'Open Sans',sans-serif;
  margin-top: calc(1.5 * var(--lendi-ui-size));
}

<Legal>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-htoDjs",
            "isStatic": false,
            "lastClassName": "c0",
            "rules": Array [
              "
  ",
              "
  box-sizing: border-box;
  line-height: 1.333;
  white-space: normal;
  border-spacing: 2px;
  text-align: left;
  word-spacing: normal;
  letter-spacing: normal;
",
              ";
  ",
              "
          background-color: ",
              [Function],
              ";
        ",
              ";
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-htoDjs",
          "target": "div",
          "toString": [Function],
          "usesTheme": false,
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="c0"
      >
        <styled.div>
          <StyledComponent
            forwardedComponent={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-dnqmqq",
                  "isStatic": false,
                  "lastClassName": "c1",
                  "rules": Array [
                    "
  ",
                    "
  box-sizing: border-box;
  max-width: 75rem;
  width: 100%;
  ",
                    "
    margin-left: ",
                    "auto",
                    ";
    margin-right: ",
                    "auto",
                    ";
  ",
                    ";
",
                    " height: 100%;
  ",
                    "
    padding: ",
                    "calc(1.5 * var(--lendi-ui-size))",
                    ";
  ",
                    ";
",
                  ],
                },
                "displayName": "styled.div",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-dnqmqq",
                "target": "div",
                "toString": [Function],
                "usesTheme": false,
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              }
            }
            forwardedRef={null}
          >
            <div
              className="c1"
            >
              <Styled(HeadingTag)
                color="shade.0"
                mt="sm"
                size="md"
              >
                <StyledComponent
                  color="shade.0"
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-bxivhb",
                        "isStatic": false,
                        "lastClassName": "c2",
                        "rules": Array [
                          "
  ",
                          "
    margin-top: ",
                          "0",
                          ";
    margin-bottom: ",
                          "0",
                          ";
  ",
                          "
  ",
                          [Function],
                          "
  ",
                          "
  ",
                          [Function],
                          "
  ",
                          [Function],
                          "
  ",
                          [Function],
                          "
  ",
                          [Function],
                          "
  ",
                          [Function],
                          "
  ",
                          [Function],
                          "
  ",
                          [Function],
                          "
",
                          ";
",
                        ],
                      },
                      "displayName": "Styled(HeadingTag)",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-bxivhb",
                      "target": [Function],
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                      Symbol(Symbol.hasInstance): [Function],
                    }
                  }
                  forwardedRef={null}
                  mt="sm"
                  size="md"
                >
                  <HeadingTag
                    className="c2"
                    color="shade.0"
                    mt="sm"
                    size="md"
                  >
                    <h3
                      className="c2"
                      color="shade.0"
                      mt="sm"
                      size="md"
                    >
                      Important legal stuff
                    </h3>
                  </HeadingTag>
                </StyledComponent>
              </Styled(HeadingTag)>
              <styled.div>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-iwsKbI",
                        "isStatic": false,
                        "lastClassName": "c3",
                        "rules": Array [
                          "
  ",
                          "
    ",
                          "
          color: ",
                          [Function],
                          ";
        ",
                          "
    ",
                          "
    ",
                          "
        font-size: calc(0.75 * var(--lendi-ui-size));
        line-height: calc(16 / 12);
      ",
                          "
    font-family: ",
                          [Function],
                          ";
  ",
                          " ",
                          "
    margin-top: ",
                          "calc(1.5 * var(--lendi-ui-size))",
                          ";
  ",
                          ";
",
                        ],
                      },
                      "displayName": "styled.div",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-iwsKbI",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="c3"
                  >
                    Lendi is the trading name of Lendi Pty Ltd (ACN 611 161 856, Credit Representative 518849), a related body corporate of Auscred Services Pty Ltd (ACN 164 638 171, Australian Credit Licence 442372). We will never sell your email address to any third party or send you nasty spam, promise.
                  </div>
                </StyledComponent>
              </styled.div>
              <styled.div>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-iwsKbI",
                        "isStatic": false,
                        "lastClassName": "c3",
                        "rules": Array [
                          "
  ",
                          "
    ",
                          "
          color: ",
                          [Function],
                          ";
        ",
                          "
    ",
                          "
    ",
                          "
        font-size: calc(0.75 * var(--lendi-ui-size));
        line-height: calc(16 / 12);
      ",
                          "
    font-family: ",
                          [Function],
                          ";
  ",
                          " ",
                          "
    margin-top: ",
                          "calc(1.5 * var(--lendi-ui-size))",
                          ";
  ",
                          ";
",
                        ],
                      },
                      "displayName": "styled.div",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-iwsKbI",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="c3"
                  >
                    Lendi is a privately owned and operated Aussie business. Our mission is to provide Aussies with the right experience when choosing a home loan from our panel of lenders including ClickLoans, a related body corporate of Auscred Services. Although Lendi compares over 1600 products from over 35 lenders, we don't cover the whole market or compare all features and there may be other features or options available to you. While Lendi is 40% owned by founders and employees, we have also been supported by some great minority shareholders including Bailador, Macquarie Bank Ltd and a number of Australian sophisticated investors. We have an independent and founder led board.
                  </div>
                </StyledComponent>
              </styled.div>
              <styled.div>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-iwsKbI",
                        "isStatic": false,
                        "lastClassName": "c3",
                        "rules": Array [
                          "
  ",
                          "
    ",
                          "
          color: ",
                          [Function],
                          ";
        ",
                          "
    ",
                          "
    ",
                          "
        font-size: calc(0.75 * var(--lendi-ui-size));
        line-height: calc(16 / 12);
      ",
                          "
    font-family: ",
                          [Function],
                          ";
  ",
                          " ",
                          "
    margin-top: ",
                          "calc(1.5 * var(--lendi-ui-size))",
                          ";
  ",
                          ";
",
                        ],
                      },
                      "displayName": "styled.div",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-iwsKbI",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="c3"
                  >
                    WARNING: This comparison rate is true only for the example given and may not include all fees and charges. Different terms, fees or other loan amounts might result in a different comparison rate. The comparison rates are based on a loan amount of $150,000 over a loan term of 25 years.
                  </div>
                </StyledComponent>
              </styled.div>
              <styled.div>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-iwsKbI",
                        "isStatic": false,
                        "lastClassName": "c3",
                        "rules": Array [
                          "
  ",
                          "
    ",
                          "
          color: ",
                          [Function],
                          ";
        ",
                          "
    ",
                          "
    ",
                          "
        font-size: calc(0.75 * var(--lendi-ui-size));
        line-height: calc(16 / 12);
      ",
                          "
    font-family: ",
                          [Function],
                          ";
  ",
                          " ",
                          "
    margin-top: ",
                          "calc(1.5 * var(--lendi-ui-size))",
                          ";
  ",
                          ";
",
                        ],
                      },
                      "displayName": "styled.div",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-iwsKbI",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="c3"
                  >
                    EXAMPLE: This example is current as at 20th October 2016. A Click Loans Online Principal and Interest Loan of $150,000 over 25 years has monthly repayments of $767. This is calculated based on the interest rate of 3.69%, comparison rate of 3.69%, upfront fees of $0 and annual fees of $0.
                  </div>
                </StyledComponent>
              </styled.div>
              <styled.div>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-iwsKbI",
                        "isStatic": false,
                        "lastClassName": "c3",
                        "rules": Array [
                          "
  ",
                          "
    ",
                          "
          color: ",
                          [Function],
                          ";
        ",
                          "
    ",
                          "
    ",
                          "
        font-size: calc(0.75 * var(--lendi-ui-size));
        line-height: calc(16 / 12);
      ",
                          "
    font-family: ",
                          [Function],
                          ";
  ",
                          " ",
                          "
    margin-top: ",
                          "calc(1.5 * var(--lendi-ui-size))",
                          ";
  ",
                          ";
",
                        ],
                      },
                      "displayName": "styled.div",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-iwsKbI",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="c3"
                  >
                    IMPORTANT INFORMATION: Loan terms of between 1 Year and 40 Years are available subject to lender and credit criteria. Maximum comparison rate will not exceed 14.99% (see comparison rate warning above). Any calculations or estimated savings do not constitute an offer of credit or a credit quote and are only an estimate of what you may be able to achieve based on the accuracy of the information provided. It doesn't take into account any product features or any applicable fees. Our lending criteria and the basis upon which we assess what you can afford may change at any time without notice. Savings shown are based on user inputted data and a loan term of 30 years. All applications for credit are subject to lender credit approval criteria.
                  </div>
                </StyledComponent>
              </styled.div>
              <styled.div>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "sc-iwsKbI",
                        "isStatic": false,
                        "lastClassName": "c3",
                        "rules": Array [
                          "
  ",
                          "
    ",
                          "
          color: ",
                          [Function],
                          ";
        ",
                          "
    ",
                          "
    ",
                          "
        font-size: calc(0.75 * var(--lendi-ui-size));
        line-height: calc(16 / 12);
      ",
                          "
    font-family: ",
                          [Function],
                          ";
  ",
                          " ",
                          "
    margin-top: ",
                          "calc(1.5 * var(--lendi-ui-size))",
                          ";
  ",
                          ";
",
                        ],
                      },
                      "displayName": "styled.div",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "sc-iwsKbI",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="c3"
                  >
                    Made with love at Circular Quay in Sydney, Australia. © 
                    <span
                      className="curYear"
                    >
                      2019
                    </span>
                    . All rights reserved.
                  </div>
                </StyledComponent>
              </styled.div>
            </div>
          </StyledComponent>
        </styled.div>
        <meta
          content="telephone=no"
          name="format-detection"
        />
      </div>
    </StyledComponent>
  </styled.div>
</Legal>
`;
